features/include/pcl/features/impl/grsd.hpp

```diff
@@ -92,21 +92,21 @@ pcl::GRSDEstimation<PointInT, PointNT, PointOutT>::computeFeature (PointCloudOut
   // Save the type of each point
   int NR_CLASS = 5; // TODO make this nicer
   std::vector<int> types (radii->points.size ());
-  for (std::size_t idx = 0; idx < radii->points.size (); ++idx)
-    types[idx] = getSimpleType (radii->points[idx].r_min, radii->points[idx].r_max);
+  std::transform(radii->points.begin (), radii->points.end (), types.begin (),
```

Comment:
Can we use const here?

```suggestion
  std::transform(radii->points.cbegin (), radii->points.cend (), types.begin (),
```